torchvision/transforms/v2/_geometry.py

```diff
@@ -26,16 +26,16 @@
 
 
 class RandomHorizontalFlip(_RandomApplyTransform):
-    """[BETA] Horizontally flip the given image/box/mask randomly with a given probability.
+    """[BETA] Horizontally randomly flip the given image/video/boxes/mask with a given probability.
```

Comment:
I should have flagged this in #7297, but it only caught my eye here. Do we really want to keep the `[BETA]` prefix, when we get a warning one line below. Seems duplicated to me:

![Screenshot from 2023-02-23 14-29-52](https://user-images.githubusercontent.com/6849766/220920770-8dfccad1-765e-47c5-996d-28ccc9aa1fea.png)

Reply:
Plus, I'm not sure if we should include 

> the given image

in the message, since that somewhat implies that you input only one of these. Maybe

```suggestion
    """[BETA] Horizontally randomly flip images, videos, bounding boxes, and mask with a given probability.
```

Reply:
The `[BETA]` header is for visibility in the listings

![image](https://user-images.githubusercontent.com/1190450/220925129-cd2eb2ca-a1a5-446e-ad33-cdbc250e5b48.png)